src/pages/workspaces/workspace/Analysis.js

```diff
@@ -0,0 +1,469 @@
+        onRename: () => setRenamingAnalysisName(name),
+        onCopy: () => setCopyingAnalysisName(name),
+        onExport: () => setExportingAnalysisName(name),
+        onDelete: () => setDeletingAnalysisName(name)
+      }))
+    )(analyses)
+
+    return div({
+      style: {
+        textAlign: 'center'
+      }
+    }, [
+      Utils.cond(
+        [_.isEmpty(analyses), () => noAnalysisBanner],
+        [!_.isEmpty(analyses) && _.isEmpty(renderedAnalyses), () => {
+          return div({ style: { fontStyle: 'italic' } }, ['No matching analyses'])
+        }],
+        [listView, () => div({ style: { flex: 1 } }, [renderedAnalyses])],
+        () => div({ style: { display: 'flex', flexWrap: 'wrap' } }, renderedAnalyses)
+      )
+    ])
+  }
+
+  // Render
+  return h(Dropzone, {
+    accept: '.ipynb, .rmd',
+    disabled: !Utils.canWrite(accessLevel),
+    style: { flexGrow: 1 },
+    activeStyle: { backgroundColor: colors.accent(0.2), cursor: 'copy' },
+    onDropRejected: () => reportError('Not a valid analysis file',
+      'The selected file is not a .ipynb notebook file or an .rmd rstudio file. Ensure your file has the proper extension.'),
+    onDropAccepted: uploadFiles
+  }, [({ openUploader }) => h(Fragment, [
+    analyses && h(PageBox, [
+      div({ style: { display: 'flex', marginBottom: '1rem' } }, [
+        div({ style: { color: colors.dark(), fontSize: 24, fontWeight: 600 } }, ['Your Analyses']),
+        h(ButtonPrimary, {
+          style: {
+            marginLeft: '6.5rem'
+          },
+          onClick: () => setCreating(true),
+          disabled: !Utils.canWrite(accessLevel),
+          tooltip: !Utils.canWrite(accessLevel) ? noWrite : undefined
+        }, [
+          div({ style: { marginBottom: '0.5rem' } }, [
+            icon('plus-circle', { style: { marginTop: '0.5rem', marginRight: '0.5rem' }, size: 21 }),
+            'Create'
+          ])
+        ]),
+        h(ButtonPrimary, {
+          style: {
+            marginLeft: '1rem'
+          },
+          onClick: openUploader,
+          disabled: !Utils.canWrite(accessLevel),
+          tooltip: !Utils.canWrite(accessLevel) ? noWrite : undefined
+        }, [
+          div({ style: { marginBottom: '0.5rem' } }, [
+            icon('upload-cloud', { style: { marginTop: '0.5rem', marginRight: '0.5rem' }, size: 21 }),
+            'Upload'
+          ])
+        ]),
+        div({ style: { flex: 2 } }),
+        !_.isEmpty(analyses) && h(DelayedSearchInput, {
+          'aria-label': 'Search analyses',
+          style: { marginRight: '0.75rem', width: 220 },
+          placeholder: 'Search analyses',
+          onChange: setFilter,
+          value: filter
+        }),
+        !_.isEmpty(analyses) && h(IdContainer, [id => h(Fragment, [
+          label({ htmlFor: id, style: { marginLeft: 'auto', marginRight: '0.75rem' } }, ['Sort By:']),
```

Comment:
`Sort By` text seems to have gotten misaligned for some reason:

![image](https://user-images.githubusercontent.com/5438223/119840593-1f73ed00-bed3-11eb-9290-f1c00758d270.png)

Reply:
I know these are not important for the time being but we tend to forget them later…would be good to change the menu items like `Most Recently Updated` to `Most recently updated` so they have the same casing as every other text on the page.

Reply:
Yeah, I didn't spend too much time here since eventually this bar will go away entirely in favor of table based sorting. See https://broadworkbench.atlassian.net/browse/IA-2700 . Also, the ticket notes that the wireframes themselves are inconsistent in how this should look, so I very much did not want to touch it this iteration (working on that in the background...) :) .